### ISSUE TITLE
[a11y] missing labels for quota select

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4468]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
The quota select should have a proper label but currently only has a `title` and placeholder.

### Actual behaviour
Either add a visual `<label>` or add `aria-label`.

<img width="1086" height="520" alt="Image" src="https://github.com/user-attachments/assets/ecdf9fd5-a1cb-4984-9c26-e1af5cbd447a" />

**Comment by codecrafting-io** [comment]
Besides that I can't set any quota, the dialog just revert to previous value
